src/main/java/reposense/model/FileTypeManager.java

```diff
@@ -0,0 +1,106 @@
+package reposense.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * {@code FileTypeManager} is responsible for holding a list of whitelisted formats and user-specified custom
+ * groupings.
+ */
+public class FileTypeManager {
+    private static final String DEFAULT_GROUP = "other";
+
+    private List<FileType> formats;
```

Comment:
Internally we are referring it as `fileTypes`?

Reply:
Data type wise, yes